.env.example

```
@@ -0,0 +1,65 @@
+APP_NAME='YokaKit'
+APP_ENV=production
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_ENV key should go before the APP_NAME key

(UnorderedKey)